### PR TITLE
use correct printf specifiers

### PR DESCRIPTION
**Pull request by slorquet**
## Summary
Remove a warning

## Impact
None

## Testing
Tested in our own board.

### CODE REVIEW

**Comment by slorquet** [comment]
This PR creates a printf format string that is longer than the allowed line length.

This is a test to see github CI reaction.

**Comment by acassis** [comment]
Hi @slorquet I think you forgot to run checkpatch before submitting your PR:

$ ./tools/checkpatch.sh -f drivers/eeprom/spi_xx25xx.c

**Comment by slorquet** [comment]
Contributing to this project is now a nightmare.

i KNOW the line is long.

How do you format a long string literal like that one?

File: drivers/eeprom/spi_xx25xx.c [hunk 1]
```diff
@@ -850,7 +850,7 @@ int ee25xx_initialize(FAR struct spi_dev_s *dev, FAR char *devname,
 
   eedev->readonly = !!readonly;
 
-  finfo("EEPROM device %s, %d bytes, %d per page, addrlen %d, readonly %d\n",
+  finfo("EEPROM device %s, %"PRIu32" bytes, %u per page, addrlen %u, readonly %d\n",
        devname, eedev->size, eedev->pgsize, eedev->addrlen, eedev->readonly);
```
Comment (gustavonihei) [inline drivers/eeprom/spi_xx25xx.c:854]:
```suggestion
  finfo("EEPROM device %s, %" PRIu32 " bytes, %u per page, addrlen %u, "
        "readonly %d\n", devname, eedev->size, eedev->pgsize,
        eedev->addrlen, eedev->readonly);
```

**Comment by gustavonihei** [comment]
> How do you format a long string literal like that one?

The C language supports the concatenation of string literals in several ways.
I've just added a suggestion of one of the possibilities.

**Comment by acassis** [comment]
> Contributing to this project is now a nightmare.
> 
> i KNOW the line is long.
> 
> How do you format a long string literal like that one?

I was going to reply you, but I saw Gustavo already did it:
You can break a string line without any issue, please take a look at arch/arm/src/cxd56xx/cxd56_sfc.c
There is a finfo("blocksize: ...." 

BTW, think twice if you think contributing code to NuttX is a nightmare, I have some drivers on Linux kernel and I can say NuttX is a "piece of cake" compared to Linux! ;-)

**Comment by slorquet** [comment]
I have others qualifiers for linux contributions but I'm not sure we want to read them here :p

Are there specific rules to split string litterals or anything that fits 80 columns is okay?

**Comment by acassis** [comment]
> I have others qualifiers for linux contributions but I'm not sure we want to read them here :p
> 
> Are there specific rules to split string litterals or anything that fits 80 columns is okay?

Basically if you don't pass the 78 column, everything is fine.

All information you need is explained here: https://cwiki.apache.org/confluence/display/NUTTX/Coding+Standard

**Comment by slorquet** [comment]
I have pushed a commit to fix the long line and tested my ability to use the rebase mechanism. This required a force push but thats okay.

**Comment by slorquet** [comment]
done...

what do I have to do next?

**Comment by acassis** [comment]
Hahaha, nothing! You are done!

**Pull request merged**